data/ReferrerWhitelist.json

```diff
@@ -88,6 +88,11 @@
             "https://*.anibis.ch/*": [
                 "https://*.google.com/*"
             ]
+        },
+        {
+            "https://source.chromium.org/*": [
+               "https://*.clients6.google.com/*"
```

Comment:
we can probably just allow *.google.com here to make sure to cover all cases, since it's all owned by Google

Reply:
Done.